iot-care-pet-example/go/README.md

```diff
@@ -0,0 +1,107 @@
+Care Pet example
+===
+
+This is an example project that demonstrates basic usage
+patterns of ScyllaDB Go driver infrastructure.
+
+The application allows tracking of pets health indicators
+and consist of 3 parts:
+
+- migrate (`/cmd/migrate`) - bootstrap database schema
+- collar sensor (`/cmd/sensor`) - accumulates pet health data and publish it
+- web app (`/cmd/server`) - REST API for tracking pets state
+
+Quick Start
+---
```

Comment:
prerequisites:
- Docker and Docker Compose
- Go

Reply:
Accepted

Reply:
Accepted